<?xml version="1.0"?>

<launch>
  <node name="data_collection" pkg="cob_people_object_detection_tensorflow" type="data_collector.py" output="screen">
    <rosparam command="load" file="$(find cob_people_object_detection_tensorflow)/launch/data_collector.yaml"/>
    <param name="output_file_name" value="$(find cob_people_object_detection_tensorflow)/data_set.csv"/>
    <param name="output_folder_name" value="$(find human_detection_and_localization_config)/datasets/"/>
  </node>
</launch>
